### PR TITLE
Add Peer JS WebRTC Tutorial to WebRTC Sidebar Macro

### PR DESCRIPTION
**Pull request by lolaodelola**
I've added my internet-connected phone tutorial to the WebRTC macro as per @chrisdavidmills recommendation. However, I've been unable to test this since the required `.env-dist.dev` & docker compose files are missing.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,4 +14,5 @@ docs
 node_modules/
 *.log
 *~
+.idea
```
Comment (chrisdavidmills) [inline .gitignore:17]: what's this?

File: macros/WebRTCSidebar.ejs [hunk 2]
```diff
@@ -79,6 +79,7 @@ var text = mdn.localStringMap({
         <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/adapter.js"><%=text['adapter_js']%></a></li>
         <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Taking_still_photos"><%=text['still_photos']%></a></li>
         <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/Simple_RTCDataChannel_sample"><%=text['simple_sample']%></a></li>
+        <li><a href="/<%=locale%>/docs/Web/API/WebRTC_API/build_a_phone_with_peerjs"><%=text['build_a_phone_with_peer_js']%></a></li>
```
Comment (chrisdavidmills) [inline macros/WebRTCSidebar.ejs:82]: We use sentence case for slugs and headings in documents, so in the first case `build_a_phone_with_peerjs` should be updated to `Build_a_phone_with_peerjs`.

Also for this to work you'll have to add an quivalent entry into the `'en-US': { ... }` object. A single line along the lines of `'build_a_phone_with_peer_js': 'Build a phone with peer js',` inside the `en-US` string list would work fine.

**Review by chrisdavidmills — APPROVED** [review]
Just a couple of comments for you to address Lola. Thanks for the addition!

Before this gets merged, we ought to move the tutorials into their correct places. Are you ready for me to review them yet?

File: .gitignore (hunk 1 above)
Comment (lolaodelola) [reply .gitignore:17]: Hey Chris, this is just some misc files generated by my IDE

File: macros/WebRTCSidebar.ejs [hunk 3]
```diff
@@ -31,6 +31,7 @@ var text = mdn.localStringMap({
     'Contribute': 'Contribute',
     'Doc_status': 'WebRTC doc status',
     'The_MDN_project': 'The MDN project',
+    'build_a_phone_with_peer_js': 'Building an internet-connected phone with Peer.js',
   },
```
Comment (lolaodelola) [inline macros/WebRTCSidebar.ejs:35]: @chrisdavidmills is this okay?

**Pull request closed without merging**